jetty9-compat-base/src/test/java/com/google/apphosting/vmruntime/jetty9/JettyRunner.java

```diff
@@ -99,17 +99,19 @@ public void waitForStarted(long timeout,TimeUnit units) throws InterruptedExcept
 
     Log.getLogger(Server.class).info("Waited!");
   }
-  
+
   @Override
   public void doStart() throws Exception {
     try {
       // find projectDir
-      File project = new File(System.getProperty("user.dir", ".")).getAbsoluteFile().getCanonicalFile();
+      File project = new File(System.getProperty("user.dir", ".")).getAbsoluteFile()
```

Comment:
+4 or better yet (if possible) wrap after the '=' sign if wrapping would fit one line.